mmcv/utils/testing.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Open-MMLab.
+
+
+def check_dict(result_dict, key_list, value_list):
+    """Check if the result_dict is correct.
+
+    Args:
+        result_dict(dict): Dict to be checked.
+        key_list(tuple): Tuple of checked keys.
+        value_list(tuple): Tuple of target values.
+
+    Returns:
+        bool: Whether the result_dict is correct.
+    """
+    for key, value in zip(key_list, value_list):
+        if result_dict[key] != value:
```

Comment:
What if the value in the dict is a Tensor? Is this supported?

Reply:
No, it cannot support it. 
But the situation, that the elements of dict are tensors, seems to be rare. Should I deal with tensor alone, or should I say the case in the comments? Or is there a better way?

Reply:
I cannot come up with the method that assert the equality of two tensors without `import torch`, unless we judge whether the comparison result is boolean type alone.

Reply:
The key issue is when `result_dict[key] != value` is a tensor or array. Since both tensor and array are Iterable, we may use all() for iterable, like below:
```python
from collections.abc import Iterable
uneq = result_dict[key] != value
uneq = all(uneq) if isinstance(uneq, Iterable) else uneq

if uneq:
    xxxxx
```

Reply:
Encountered some problems, using a less elegant way